components/dash-table/src/dash-table/utils/TableClipboardHelper.ts

```diff
@@ -85,6 +85,7 @@ export default class TableClipboardHelper {
         includeHeaders: boolean
     ): {data: Data; columns: Columns} | void {
         const text = Clipboard.get(ev);
+        console.log(text);
```

Comment:
🔪 Let's take this debugging statement out before merging

Reply:
Fixed!